crates/control/src/behavior/dribble.rs

```diff
@@ -345,3 +364,20 @@ fn ball_is_close_to_own_goal(
     let goal_to_ball = ball_position - own_goal_center;
     goal_to_ball.norm() < is_close_threshold
 }
+
+fn is_ball_in_opponents_corners(
+    ball_position: &Point2<f32>,
+    parameters: &FindKickTargets,
+    field_dimensions: &FieldDimensions,
+    robot_to_field: Isometry2<f32>,
+) -> bool {
+    let global_ball = robot_to_field * ball_position;
+    let left_opponent_corner = point![field_dimensions.length / 2.0, field_dimensions.width / 2.0];
+    let right_opponent_corner =
+        point![field_dimensions.length / 2.0, -field_dimensions.width / 2.0];
+    let ball_near_left_opponent_corner = distance(&global_ball, &left_opponent_corner)
+        < parameters.distance_from_corner;
+    let ball_near_right_opponent_corner = distance(&global_ball, &right_opponent_corner)
+        < parameters.distance_from_corner;
+    ball_near_left_opponent_corner || ball_near_right_opponent_corner
```

Comment:
some `.any()`?

Reply:
Could you elaborate? 🤔 These are two booleans. Do you suggest to use an array to be able to call `.any()` on it?

Reply:
IMHO, it is clear enough without arrays and would only get more complex. It also fits the style of the rest of the file.